CONTRIBUTING.md

```diff
@@ -235,3 +238,12 @@ directory. As an expedient to make the third_party code work well with our build
 code in third_party should be given a stub pubspec.yaml file so that you can
 reference the resources from the packages directory from
 `packages/devtools_app/web/index.html`
+
+## Enabling and activating DCM
+
+1. Install the executable for your target platform. You can refer to [this guide](https://dcm.dev/docs/teams/getting-started/#installation).
+2. [Get the license key](http://goto.google.com/dart-devtools-dcm-keys) and activate DCM. To do so, run `dcm activate --license-key=YOUR_KEY` from the console.
+3. Install the extension for your IDE. If you use VS Code, you can get it from [the marketplace](https://marketplace.visualstudio.com/items?itemName=dcmdev.dcm-vscode-extension). If you use IntelliJ IDEA or Android Studio, you can find the plugin [here](https://plugins.jetbrains.com/plugin/20856-dcm).
+4. Reload the IDE.
```

Comment:
Let's add instructions for how to run from the command line as well (if someone is using the free tier, they can't use the IDE plugins right?)

Reply:
The problem is that the Teams version provides more rules and simply running the free version from the console might give a false confidence that everything is fine. So the difference is more than just in the IDE integrations.

Reply:
Do you still want to mention this?